<ui version="4.0" >
 <class>KeyboardInputActionWidget</class>
 <widget class="QWidget" name="KeyboardInputActionWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="KTextEdit" name="input" />
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Window</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="active_radio" >
          <property name="text" >
           <string>Active Window</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="specific_radio" >
          <property name="text" >
           <string>Specific Window</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="action_radio" >
          <property name="text" >
           <string>Action Window</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="WindowDefinitionListWidget" native="1" name="windowdef_list" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>WindowDefinitionListWidget</class>
   <extends>QWidget</extends>
   <header>helper_widgets/window_definition_list_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
